src/vs/workbench/parts/debug/electron-browser/statusbarColorProvider.ts

```diff
@@ -62,10 +68,24 @@ export class StatusBarColorProvider extends Themable implements IWorkbenchContri
 			return STATUS_BAR_NO_FOLDER_BACKGROUND;
 		}
 
-		// debugging
+		// Debugging
 		return STATUS_BAR_DEBUGGING_BACKGROUND;
 	}
 
+	private getForegroundColorKey(): string {
```

Comment:
This looks almost identical to `getBackgroundColorKey`. Can we just have one method where all color keys are passed in? Something like `getColorKey(noFolderColor, debuggingColor, normalColor)`?